### PR TITLE
New Contributor

### PR DESCRIPTION
**Pull request by Rajputusman**
closes: #1231 

Added Ananda as Meme Designer in Marketing Team with contributor ID 71

### CODE REVIEW

**Comment by thesanjeevsharma** [comment]
@Rajputusman Can you update this PR to only add one contributor. I have created new issues for other contributors. 

Also, ID 70 is in use now. You should be using 71, 72, and 73. Confirm once, though.

File: src/data/teams.json [hunk 1]
```diff
@@ -831,6 +831,45 @@
...
           "slackUsername": "Chibuzor"
         }
+      },
+      {
+        "title": "Meme Designer",
+        "isLead": false,
+        "payPerDay": 2800,
+        "createdDate": "2020-07-08T02:14:59.307535Z",
+        "contributor": {
+          "contributorId": "70",
```
Comment (thesanjeevsharma) [inline src/data/teams.json:841]: ID in use already.

**Review by thesanjeevsharma — CHANGES_REQUESTED** [review]
The issue has been split into 3 different issues.

**Comment by Rajputusman** [comment]
@thesanjeevsharma no problem on it

**Review by thesanjeevsharma — APPROVED** [review]

**Pull request merged**